terraform/modules/check-user-expiry-lambda/lambda/lambda_function.py

```diff
@@ -66,11 +72,22 @@ def query_dynamodb_users_about_expire():
     return response["Items"]
 
 
-def process_items(items):
+def query_dynamodb_users_expired_today():
+    print("Query dynamodb table user for users expiring today")
+    today = datetime.date.today()
+    response = table.scan(
+        FilterExpression=Attr("expiration_date").eq(
```

Comment:
Looking at expiration dates in the DB, they are datetimes, so will they ever be equal to 'today'? Does str(today) display it in the same ISO8601 format as its dynamo stores dates as Strings?